core/state_processor.go

```diff
@@ -93,6 +93,20 @@ func (p *StateProcessor) Process(block *types.Block, statedb *state.StateDB, cfg
 				return nil, nil, 0, fmt.Errorf("Block contains transaction with receiver in black-list: %v", tx.To().Hex())
 			}
 		}
+		// validate minFee slot for TomoZ
+		if tx.IsTomoZApplyTransaction() {
+			copyState := statedb.Copy()
+			if err := ValidateTomoZApplyTransaction(p.bc, copyState, common.BytesToAddress(tx.Data()[4:])); err != nil {
```

Comment:
seems need to check hardfork for this validation

Reply:
should we use this one ?https://github.com/tomochain/tomochain/blob/master/common/constants.go#L34

Reply:
updated !
https://github.com/tomochain/tomochain/pull/236/commits/ceed05816a6058da33d1450bd0af05e03667f5e5